docs/source/athena-deploy.rst

```diff
@@ -10,7 +10,7 @@ The name of the Lambda function is Athena Partition Refresh.
 
 The purpose of this function is to periodically refresh Athena tables to enable the search-ability of data.
 
-The default refresh interval is 10 minutes.
+The default refresh interval is 10 minutes, this can be adjusted based on throughput.
```

Comment:
Add any context the user should reason about when trying to decide what # they should pick. Example: Why wouldn't everyone want to just put this to 1 minute in order to have data available the quickest?